### PR TITLE
Add example for each section of `pyproject.toml`

### PR DESCRIPTION
**Pull request by gunungpw**
# Pull Request Check List

Resolves: #issue-number-here -> not found maybe I use wrong keyword :)

- [x] Added **tests** for changed code. -> None
- [x] Updated **documentation** for changed code. -> None

Hello, thank you for Poetry project. One of papercut in poetry docs was examples for simple section in `pyproject.toml` like `authors`, `maintainers` need to use `list` and `<>` to point email. For beginner like me, this example will help them to more understand how to fill the `pyproject.toml`.
Also another questions how to write multiple maintainer?

```toml
maintainer = [
    "Apple Juice <apple@juice.org>",
    "Orange Juice <orange@juice.org>",
]
```
Did example right, do I need use trailing comma, how about whitespace do toml use `space` or `tabs`. Maybe  I need to read toml spec ;)

### CODE REVIEW

File: docs/pyproject.md [hunk 1]
```diff
@@ -61,40 +73,76 @@ More identifiers are listed at the [SPDX Open Source License Registry](https://s
...
+```
+
 ## authors
 
 The authors of the package. **Required**
 
 This is a list of authors and should contain at least one author. Authors must be in the form `name <email>`.
 
+```toml
+license = [
```
Comment (dimbleby) [inline docs/pyproject.md:87]: oops

Comment (gunungpw) [reply docs/pyproject.md]: my bad :)

File: docs/pyproject.md [hunk 2]
```diff
@@ -61,40 +73,76 @@ More identifiers are listed at the [SPDX Open Source License Registry](https://s
...
+
 ## authors
 
 The authors of the package. **Required**
 
 This is a list of authors and should contain at least one author. Authors must be in the form `name <email>`.
 
+```toml
+authors = [
+    "Sébastien Eustace <sebastien@eustace.io>"
```
Comment (neersighted) [inline docs/pyproject.md:88]: Please add a trailing comma -- it's generally TOML best practice when you split a list over multiple lines like this.

File: docs/pyproject.md [hunk 3]
```diff
@@ -61,40 +73,76 @@ More identifiers are listed at the [SPDX Open Source License Registry](https://s
...
+
 ## maintainers
 
 The maintainers of the package. **Optional**
 
 This is a list of maintainers and should be distinct from authors. Maintainers may contain an email and be in the form `name <email>`.
 
+```toml
+maintainers = [
+    "Richard Brave <email@example.org>"
```
Comment (neersighted) [inline docs/pyproject.md:100]: Ditto

File: docs/pyproject.md [hunk 4]
```diff
@@ -34,6 +42,10 @@ If you would like to use semantic versioning for your project, please see
 
 A short description of the package. **Required**
 
+```toml
+description = "The description of the package"
```
Comment (neersighted) [inline docs/pyproject.md:46]:
```suggestion
description = "A short description of the package."
```

**Review by neersighted — CHANGES_REQUESTED** [review]

**Comment by gunungpw** [comment]
Okay issue have been fix :)

**Review by neersighted — APPROVED** [review]

**Pull request merged**